react-app/src/library/services/vendor-implementations/sennheiser/sennheiser.ts

```diff
@@ -285,6 +292,10 @@ export default class SennheiserService extends VendorImplementation {
         break;
       case SennheiserEvents.CallEnded:
         if (payload.EventType === SennheiserEventTypes.Notification) {
+          this._sendMessage({
+            Event: SennheiserEvents.UnmuteFromApp,
+            EventType: SennheiserEventTypes.Request,
+          });
```

Comment:
Oddly enough, I found out through my testing that EPOS/Senn doesn't really have a way to end calls via the device when on hold.  So I ommitted the "hold reset" here because it _shouldn't_ be an issue